README.md

```diff
@@ -1,116 +1,111 @@
-    mesh_min: 10,22         # x / y offsets for cartographer.
-    mesh_max: 290,280       # add a little space from the back of the bed to prevent scanning screws or crashing into the motor mounts
-    probe_count: 40,40      # tested up to 150x150 points, any higher will timeout the mcu after meshing.
-    algorithm: bicubic      # required for above 5x5 meshing
-    bicubic_tension: 0.1
-    ```
+7. In `printer.cfg` under `[stepper_z]` edit the endstop pin to the following:
 
+   1. Remove the following:
+   ``` ini
+   endstop_pin: tmc2209_stepper_z:virtual_endstop# PA15   #probe:z_virtual_endstop 
+   position_endstop: 0
+   ```
+   2. Replace with the following:
+   ``` ini
+   endstop_pin: probe:z_virtual_endstop # use cartographer as virtual endstop
+   homing_retract_dist: 0 # cartographer needs this to be set to 0
+   ```
+8. Inside `printer.cfg` remove your `[bed_mesh]` section and replace it with the appropriate section for your printer:
+   * For the K1:
+   ``` ini
+   [bed_mesh]              # K1
+   zero_reference_position: 112,112
+   speed: 135              # recommended max 150 - absolute max 180. Going above 150 will cause mcu hanging / crashing or inconsistent spikey meshes due to bandwidth limitation.  
+   mesh_min: 30,25         # up to 30x30 if you have a weird spike bottom left of mesh
+   mesh_max: 210,210       # 210 max before hitting rear plate screws on stock bed
+   probe_count: 20,20      # tested 100x100 working
+   algorithm: bicubic      # required for above 5x5 meshing
+   bicubic_tension: 0.1
+   ```
+   * For the K1 Max:
+   ``` ini
+   [bed_mesh]              # K1 MAX
+   speed: 150              # max of 150 or cartographer will stutter / timeout
+   mesh_min: 10,22         # x / y offsets for cartographer.
+   mesh_max: 290,280       # add a little space from the back of the bed to prevent scanning screws or crashing into the motor mounts
+   probe_count: 40,40      # tested up to 150x150 points, any higher will timeout the mcu after meshing.
+   algorithm: bicubic      # required for above 5x5 meshing
+   bicubic_tension: 0.1
+   ```
 
 # First Steps and Calibration:
-1. Move your bed plate 2-3 mm away from the nozzle 
-2. On the homescreen of your web UIX,
-    1. press the ```CARTO_CALIBRATE``` macro and wait for the Z offset wizard to pop up.
-    2. Follow the [Paper Test Method](https://www.klipper3d.org/Bed_Level.html#the-paper-test) 
-    3. Upon completion ```SAVE_CONFIG```
- 
-**IMPORTANT SAFETY CHECK**
 
-4. While your motors are disabled manually move the bed away from the nozzle (at least a fist away) and type into klipper’s console: ```M119```
+1. Move your bed plate 2-3 mm away from the nozzle 
+2. On the homescreen of your web UIX, press the **`CARTO_CALIBRATE`** macro and wait for the Z offset wizard to pop up.
+Follow the [Paper Test Method](https://www.klipper3d.org/Bed_Level.html#the-paper-test)
+Upon completion ```SAVE_CONFIG```
+    **IMPORTANT SAFETY CHECK**
```

Comment:
```suggestion
3. **IMPORTANT SAFETY CHECK**<br/>
   While your motors are disabled manually move the bed away from the nozzle (at least a fist away) and type into klipper’s console: ```M119```
If your Z endstop is “OPEN” you are safe to continue; however, if it is “TRIGGERED” re-do step 2 or begin troubleshooting.
```

Reply:
How about this change to the safety check instead; put it at the top of step so that we don't interrupt the list.

Reply:
The whole goal of the safetycheck was to interrupt the list and make the user STOP and check

Reply:
So the check should be structured as an independent process rather than a third step?

If thats the case, we could do two 'before' and 'after' lists. If we want the 'after' list to start at step 3, we'll have to structure those steps as HTML rather than markdown (as you can't signal you want an `<ol start=3>` via Markdown syntax)

Reply:
I had tinkered a little with using html style OLs but it looked REALLY bad on th markdown formatting (imho) 

maybe we do a sublist after the safety check so it is ordered more logically?

Reply:
I committed a proposal - a subsection for this as a trade-off of clean markdown and properly rendered HTML. Does this look like what you are going for?

Reply:
> I committed a proposal - a subsection for this as a trade-off of clean markdown and properly rendered HTML. Does this look like what you are going for? https://github.com/K1-Klipper/cartographer-klipper/blob/88c47de17ca0e96a32a0a077e168d4c2cc8e6126/README.md#first-steps-and-calibration

if @tlace17  thinks its good i think its good